ceph-salt-formula/salt/ceph-salt/common/sshkey.sls

```diff
@@ -14,10 +14,9 @@ create ssh user:
       - users
     - failhard: True
 
-install sudo:
-  pkg.installed:
-    - pkgs:
-      - sudo
+sudo sanity:
```

Comment:
On `ceph-salt apply`, this will be executed before being installed by `software.sls`, which may cause `ceph-salt apply` to always fail (until `sudo` is manually installed):

https://github.com/ceph/ceph-salt/blob/master/ceph-salt-formula/salt/ceph-salt/apply/init.sls#L5-L10

Reply:
That sucks. Is there a way to check for the existence of this file in the Jinja rendering stage? Then we could run pkg.installed only if it's actually needed.